From: Wouter Verhelst <w@uter.be>
Date: Tue, 1 Dec 2015 13:59:27 +0100
Subject: [PATCH] Don't make the readit() function depend on WORDS_BIGENDIAN

We had accidentally moved the readit() function to the wrong location in
this cliserv file, which meant that it wouldn't be compiled in if
WORDS_BIGENDIAN was defined.

Fix that.
---
 cliserv.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/cliserv.c b/cliserv.c
index 93835f9800af..241aa556ed26 100644
--- a/cliserv.c
+++ b/cliserv.c
@@ -81,6 +81,7 @@ u64 ntohll(u64 a) {
 	hi = ntohl(hi);
 	return ((u64) lo) << 32U | hi;
 }
+#endif
 
 /**
  * Read data from a file descriptor into a buffer
@@ -103,5 +104,3 @@ void readit(int f, void *buf, size_t len) {
 		}
 	}
 }
-
-#endif
